.github/workflows/publish-docs.yml

```diff
@@ -15,24 +15,26 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 1000  # should be enough to reach the most recent tag
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Install
+      shell: bash -l {0}
+      run: source continuous_integration/scripts/install.sh
+    - name: Install documentation-building requirements
       run: |
-        sudo apt install graphviz procserv libhdf5-dev
-        # These packages are installed in the base environment but may be older
-        # versions. Explicitly upgrade them because they often create
-        # installation problems if out of date.
-        python -m pip install --upgrade pip setuptools numpy
-        pip install .
-        pip install -r requirements-docs.txt
-        pip list
+        python -m pip install -r requirements-docs.txt
+        python -m pip list
     - name: Build Docs
       run: |
-        git fetch --tags
         make -C docs/ html
+    - name: Set env.REPOSITORY_NAME  # just the repo, as opposed to org/repo
```

Comment:
Why not do this very early on?  It becomes available earlier.  Some other repos may want this for an action.